### ISSUE TITLE
Output.php __contructor with syntax error

### ISSUE THREAD

**Issue by fracolo** [symfony/symfony#34492]
**Symfony version(s) affected**: 4.*

**Description**  

During composer install, while executing "Executing script security-checker security:check"
the following message occurs : 

```
Script security-checker security:check returned with error code 255
!!  PHP Parse error:  syntax error, unexpected '?', expecting variable (T_VARIABLE) in APP_PATH/vendor/symfony/console/Output/Output.php on line 40
!!  

```
**How to reproduce**  

**Possible Solution**  

Since 4.0, the __constructor() function in the file symfony/console/Output/Output.php has the first parameter declared as "?int $verbosity" which would be, I guess, "int $verbosity".
**Additional context**

**Comment by dmaicher** [comment]
Seems you are using php < 7.1? You need at least 7.1 😉

**Comment by fracolo** [comment]
I have 7.0, 7.1, 7.2 and 7.3 php versions available and ran the composer script with php 7.3 . So, It seems that somewhere, the default "php" command name is called (the only possibility for 7.0 version to be called.
composer.phar has been build with php7-3 command and all packages depencies relies on this version.
Where the "calling" php version is beeing lost ?

**Comment by chalasr** [comment]
> Where the "calling" php version is beeing lost ?

I'd say somewhere around https://github.com/composer/composer/blob/master/src/Composer/Command/RunScriptCommand.php and https://github.com/composer/composer/blob/master/src/Composer/Util/ProcessExecutor.php.
This depends on your multiversion PHP setup, we won't be able to debug the situation better than you. 

I'm going to close this issue as it does not seem to be a bug in Symfony itself.
Thank you.
